go/saltpack/const.go

```diff
@@ -39,6 +39,22 @@ const EncryptionArmorHeader = "BEGIN KEYBASE ENCRYPTED MESSAGE"
 // armored KB message.
 const EncryptionArmorFooter = "END KEYBASE ENCRYPTED MESSAGE"
 
+// SignedArmorHeader is the header that marks the start of signed
+// armored KB message.
+const SignedArmorHeader = "BEGIN KEYBASE SIGNED MESSAGE"
```

Comment:
I prefer mixed case.

BTW, these should probably be added to the spec as there is no way to know the "right" text.